### PR TITLE
Add new `RequestMethod` format

### PR DESCRIPTION
**Pull request by ArthurMa1978**

### CODE REVIEW

File: test/TestProjects/MgmtRenameRules/readme.md [hunk 1]
```diff
@@ -48,4 +48,6 @@ rename-mapping:
   UpgradeOperationHistoricalStatusInfo.type: -|resource-type # this configuration does not change its property name, only changes its format to resource-type
   DiskSecurityTypes.ConfidentialVM_VMGuestStateOnlyEncryptedWithPlatformKey: ConfidentialVmGuestStateOnlyEncryptedWithPlatformKey
   AdditionalUnattendContent.backupFrequency: -|integer
+  AdditionalUnattendContent.method: requestMethod|request-method
+  AdditionalUnattendContent.basePath: basePathUri|uri
```
Comment (Yao725) [inline test/TestProjects/MgmtRenameRules/readme.md:52]: Does `AdditionalUnattendContent` contain these properties?

Comment (fengzhou-msft) [inline test/TestProjects/MgmtRenameRules/readme.md:52]: method and basePath belong to DataDisk.

Comment (ArthurMa1978) [reply test/TestProjects/MgmtRenameRules/readme.md]: fixed

Comment (ArthurMa1978) [reply test/TestProjects/MgmtRenameRules/readme.md]: fixed

**Review by fengzhou-msft — APPROVED** [review]

**Pull request merged**